src/Emulator/Main/Peripherals/Timers/ComparingTimer.cs

```diff
@@ -49,7 +49,7 @@ public ComparingTimer(IClockSource clockSource, long frequency, IPeripheral owne
             InternalReset();
         }
 
-        protected ComparingTimer(IClockSource clockSource, long frequency, ulong limit = ulong.MaxValue, Direction direction = Direction.Ascending, bool enabled = false, WorkMode workMode = WorkMode.OneShot, bool eventEnabled = false, ulong compare = ulong.MaxValue, uint divider = 1) 
```

Comment:
Formatting fixes are always welcomed, but please keep them in a separate commit.